CMake Error at /opt/ros/galactic/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
  execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
  interfaces --arguments-file
  /home/prakt3/praktikum_ws/src/whiteboard_drawing_script/interfaces/build/interfaces/rosidl_adapter__arguments__interfaces.json
  --output-dir
  /home/prakt3/praktikum_ws/src/whiteboard_drawing_script/interfaces/build/interfaces/rosidl_adapter/interfaces
  --output-file
  /home/prakt3/praktikum_ws/src/whiteboard_drawing_script/interfaces/build/interfaces/rosidl_adapter/interfaces.idls)
  returned error code 1:

  Error processing 'PointTranslation3D pointTranslation' of
  'interfaces/Getpose_Response': ''pointTranslation' is an invalid field
  name.  It should have the pattern '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''

  Traceback (most recent call last):

    File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
      return _run_code(code, main_globals, None,
    File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
      exec(code, run_globals)
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
      sys.exit(main())
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
      abs_idl_file = convert_to_idl(
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
      return convert_srv_to_idl(
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
      srv = parse_service_string(package_name, input_file.stem, content)
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 856, in parse_service_string
      response_message = parse_message_string(
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 515, in parse_message_string
      fields.append(Field(
    File "/opt/ros/galactic/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 353, in __init__
      raise NameError(

  NameError: 'pointTranslation' is an invalid field name.  It should have the
  pattern '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'

Call Stack (most recent call first):
  /opt/ros/galactic/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
  CMakeLists.txt:27 (rosidl_generate_interfaces)


